src/fides/api/api/v1/endpoints/user_endpoints.py

```diff
@@ -549,6 +549,8 @@ def user_login(
             username=config.security.root_username,
             created_at=datetime.utcnow(),
         )
+
+        logger.warning("Successful root user login!")
```

Comment:
```suggestion
        logger.warning("Root Username & Password were used to login! If unexpected, review security settings (FIDES__SECURITY__ROOT_USERNAME and FIDES__SECURITY__ROOT_PASSWORD)")
```

Reply:
hmm, this doesn't feel a bit noisy for every root user login?

Reply:
ok, understood now that we want to provide extra context to prevent mis-reading the log as a concern in cases when it's totally expected 👍